testsuite/ui/views/auth.py

```diff
@@ -20,7 +20,7 @@ class Auth0View(View, Navigable):
 
     def login(self, email, password):
         """Login to 3scale via Auth0"""
-        self.last_login_button.wait_displayed()
+        self.last_login_button.wait_displayed(timeout="20s")
```

Comment:
Add comment please why it is needed. One day someone else will be looking at code, without context might be convinced it is unnecessary delay.